bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/Higgs/thq_4f_ckm_LO_ctcvcp_MH/thq_4f_ckm_LO_ctcvcp_MH125_customizecards.dat

```diff
@@ -0,0 +1,4 @@
+#put card customizations here (change top and higgs mass for example)
+set param_card mass 6 172.5
+set param_card mass 25 125
+set param_card yukawa 6 -172.5
```

Comment:
you set the coupling to be negative, is it a typo

Reply:
for this one,it was used in the same way in Run2 if I understand correctly: https://github.com/cms-sw/genproductions/blob/8d97961cda6fffccfa09cc245b715ce6f2b43f6f/bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/Higgs/thq_4f_ckm_LO_ctcvcp_MH/thq_4f_ckm_LO_ctcvcp_MH_customizecards.dat, Should we set it positive?

Reply:
I'm not sure about it, it's better to confirm with the analyzer or the person who made the initial PR for the motivation of this setup

Reply:
@sihyunjeon I checked the model HC_NLO_X0_UFO-v1.3.zip, it's 
ymt = Parameter(name = 'ymt',
                nature = 'external',
                type = 'real',
                value = 164.5,
                texname = '\\text{ymt}',
                lhablock = 'YUKAWA',
                lhacode = [ 6 ])
also I don't understand the meaning of a negative yukawa.